[OSDOCS#17136]: Changes as per CQA guidelines (#111163)
openshift/openshift-docs: snippets/microshift-healthy-pods-snip.adoc

[NOTE]
====

> This example output shows a basic {microshift-short} installation. If you installed optional RPMs, the status of pods running those services also appears in the output.

Reviewer: ```suggestion
This example output shows a basic {microshift-short} installation. If you installed optional RPMs, the status of pods running those services is displayed in the output.
```
style rule
